packages/react-storage/src/components/StorageBrowser/context/elements/__tests__/IconElement.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { IconElement, IconVariant, DEFAULT_ICON_PATHS } from '../IconElement';
+
+describe('IconElement', () => {
+  it('should render icon', () => {
+    const { container } = render(<IconElement />);
+
+    expect(container).toBeInTheDocument();
+  });
+
+  it('should apply default icon attributes', () => {
+    const { container } = render(<IconElement />);
+
+    const svgElement = container.querySelector('svg');
+
+    expect(svgElement).toHaveAttribute('aria-hidden', 'true');
+    expect(svgElement).toHaveAttribute('width', '24');
+    expect(svgElement).toHaveAttribute('height', '24');
+    expect(svgElement).toHaveAttribute('viewBox', '0 -960 960 960');
+    expect(svgElement).toHaveAttribute('fill', 'none');
+    expect(svgElement).toHaveAttribute('xmlns', 'http://www.w3.org/2000/svg');
+  });
+
+  it('should render correct path for each icon variant', () => {
```

Comment:
Prefer jest idiom [`it.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) instead of loops inside test bodies:
```ts
    it.each(Object.values(DEFAULT_ICON_PATHS))('should render correct path for a %s `Icon` variant', ([variant, expectedPath]) => {
      const { container } = render(<IconElement variant={variant} />);

      const path = container.querySelector('path');

      expect(path).toBeInTheDocument();
      expect(path).toHaveAttribute('d', expectedPath);
    })
```